packages/gatsby-plugin-image/src/components/__tests__/gatsby-image.browser.tsx

```diff
@@ -19,8 +19,9 @@ describe(`GatsbyImage browser`, () => {
 
   beforeEach(() => {
     console.warn = jest.fn()
-    ;(global as GlobalOverride).SERVER = true
```

Comment:
Is the GlobalOverride from above now unused/obsolete?

Reply:
Yes! Good spot. Removed.